From 11f3931ff151b0c02498212abc0f021264e7652b Mon Sep 17 00:00:00 2001
From: Ido Yariv <ido@wizery.com>
Date: Wed, 6 Apr 2011 00:31:23 +0300
Subject: [PATCH 7/8] wl12xx: Fix out-of-tree build issues

The wl12xx can be compiled out of tree (e.g. using compat-wireless).

The wl12xx platform data functions should be selected for the relevant
platforms, even if the builtin driver is not built.

Also, since cfg80211/mac80211 might also be built externally, select
WIRELESS_EXT when selecting WL12XX_PLATFORM_DATA, to enable wext.
Signed-off-by: Ido Yariv <ido@wizery.com>
---
 drivers/net/wireless/wl12xx/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/linux-2.6.37-psp04.04.00.01/drivers/net/wireless/wl12xx/Kconfig b/linux-2.6.37-psp04.04.00.01/drivers/net/wireless/wl12xx/Kconfig
index b447559..aeff826 100644
--- a/linux-2.6.37-psp04.04.00.01/drivers/net/wireless/wl12xx/Kconfig
+++ b/linux-2.6.37-psp04.04.00.01/drivers/net/wireless/wl12xx/Kconfig
@@ -42,5 +42,5 @@ config WL1271_SDIO
 
 config WL12XX_PLATFORM_DATA
 	bool
-	depends on WL1271_SDIO != n
 	default y
+	select WIRELESS_EXT
-- 
1.7.4.1

